pages/_app.tsx

```diff
@@ -5,6 +5,9 @@ import { Bounce, ToastContainer } from 'react-toastify';
 import 'react-toastify/ReactToastify.css';
 import { useEffect, useState } from 'react';
 import { jost } from '~/utils/fonts';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+
+const queryClient = new QueryClient();
```

Comment:
En tiedä onko sulla kokemusta Tanstackin kanssa. Onko tämä ihan ok paikka laittaa `QueryClient`. 

Googlen Gemini vastasi seuraavasti:
`Yes, you can definitely set the QueryClient within your _app.tsx file in a Next.js application. This is a common approach, as it allows you to initialize the client once and share it across all pages in your application.`
joten lienee ihan ok

Reply:
Jep tämä on juuri oikea paikka 👍